source/Octopus.Tentacle.Client/Scripts/ScriptServiceVersion.cs

```diff
@@ -1,6 +1,7 @@
 ﻿namespace Octopus.Tentacle.Client.Scripts
 {
-    record ScriptServiceVersion(string Value)
+    // TODO should not be public.
```

Comment:
I assume we will address this in a future

Reply:
This will be addressed as part of the work to use a serialized version of command context.

But yes, we shouldn't be leaving these TODOs.